graphql_compiler/tests/integration_tests/test_backends_integration.py

```diff
@@ -12,31 +12,60 @@
 from ...tests import test_backend
 from ...tests.test_helpers import generate_schema, generate_schema_graph
 from ..test_helpers import SCHEMA_TEXT, compare_ignoring_whitespace, get_schema
-from .integration_backend_config import MATCH_BACKENDS, SQL_BACKENDS
+from .integration_backend_config import (
+    MATCH_BACKENDS, NEO4J_BACKENDS, REDISGRAPH_BACKENDS, SQL_BACKENDS
+)
 from .integration_test_helpers import (
-    compile_and_run_match_query, compile_and_run_sql_query, sort_db_results
+    compile_and_run_match_query, compile_and_run_neo4j_query, compile_and_run_redisgraph_query,
+    compile_and_run_sql_query, sort_db_results
 )
 
 
-# Store the test parametrization for running against all backends. Individual tests can customize
-# the list of backends to test against with the full @parametrized.expand([...]) decorator.
-all_backends = parameterized.expand([
+all_backends_list = [
     test_backend.ORIENTDB,
     test_backend.POSTGRES,
     test_backend.MARIADB,
     test_backend.MYSQL,
     test_backend.SQLITE,
     test_backend.MSSQL,
-])
+    test_backend.NEO4J,
+    test_backend.REDISGRAPH,
+]
+
+# Store the test parametrization for running against all backends. Individual tests can customize
```

Comment:
[From](https://github.com/kensho-technologies/graphql-compiler/pull/369#discussion_r303974011) PR #369:

> I'm hesitant to have two definitions. Also, almost_all_backends with no arguments seems like bad design -- it doesn't communicate that it actually doesn't exclude any backends. Why not have one definition, all_backends and allow an exclude to be passed to it? Then, if there are no args, it does the obvious thing, and if it's called with a kwarg called e.g. except_backends then it also does the obvious thing?

Sounds good to me. How about renaming `all_backends` -> `use_all_backends`? Then the decorator might look like `@use_all_backends(except_backends([test_backend.REDISGRAPH]))` which almost looks like natural language to me. We could also replace `except_backends` with `except_for_backends` in that case.

Reply:
`use_all_backends` sounds fine to me. I'd make `except_backends` be a kwarg instead of a function of its own, so the call looks like `@use_all_backends(except_backends=[test_backend.REDISGRAPH])`.

Reply:
done-- now, if we use the decorator with no arguments it's written as `@use_all_backends()` which might look a bit weird, but the alternative (just `@use_all_backends` if there are no arguments) requires a [complicated structure](https://realpython.com/primer-on-python-decorators/#both-please-but-never-mind-the-bread) that probably wouldn't be very easy to read later on.